docs/source/faq.rst

```diff
@@ -34,3 +34,21 @@ measurements, for matching. Since there are two VLP-16s, 160 UDP packets is
 enough for roughly 2 revolutions, one per VLP-16.
 
 __ https://github.com/googlecartographer/cartographer_ros/blob/master/cartographer_ros/configuration_files/backpack_3d.lua
+
+Why is IMU data required for 3D SLAM, but not for 2D?
+-----------------------------------------------------
+
+In 2D, Cartographer supports running the correlative scan matcher that is used for finding loop closure constraints also for local SLAM.
```

Comment:
matcher, which is normally used for finding loop closure constraints, for local SLAM.